Tests/src/DieTest.java

```diff
@@ -0,0 +1,47 @@
+
+import GamePackage.Die;
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/*****************************************************************
+ JUnit tests for the Die Class
+
+ @author Dylan Kernohan
+ @version Winter 2018
+ *****************************************************************/
+public class DieTest {
+
+    Die die = new Die();
+
+    /*****************************************************************
+     Test the setValue method
+     *****************************************************************/
+    @Test
+    public void setValueTest(){

```

Comment:
In keeping with my previous comment, do we also want to test that the Die class doesn't accept a set value input that isn't in the range of 1-6?

Reply:
I don't think you should write your own test. As discussed in class, this might make it hard and a bit bias, to make sure you properly write the test. I also think the test should be part of a separate issue.